### DISCUSSION TITLE
CodeIgniter website issue

### DISCUSSION THREAD

**Discussion by Nick201986** [bcit-ci/CodeIgniter#6222] [General]
Hi I have a website nulearn.in

It is working fine in local host, when I upload on server. It is giving error 

This page isn’t working www.nulearn.in is currently unable to handle this request.
HTTP ERROR 500

**Comment by Oscabrera** [comment]
I'm glad you were able to find the bug that was holding up your launch, it would be great if you shared your lessons learned.
